### ISSUE TITLE
Multithreading

### ISSUE THREAD

**Issue by ghost** [jellysquid3/Lithium#146]
The title is self-explaining, threading may increase performance on lot's of things

**Comment by ZtereoHYPE** [comment]
Hey, this has already been discussed before on another issue asking for a similar thing.

I'm simply gonna redirect you to issue #100 so you can have a good read and get a little bit more informed.

**Comment by ghost** [comment]
I just read that issue, but the reason I've asked for feature is a forge mod did it for 1.16 and I thought it'll be possible to implement but I never considered possible bugs or glitches since I never worked on any modding project. In a normal coding project, I can easily implement with simple changes in code. Sorry if I talked like I'm offensive, I'm just curious about more technical details.

**Comment by Mhowser** [comment]
Please provide a link to the mod

**Comment by ghost** [comment]
https://www.curseforge.com/minecraft/mc-mods/mcmt-multithreading

**Comment by Mhowser** [comment]
The source code: https://github.com/jediminer543/JMT-MCMT

**Comment by AMereBagatelle** [comment]
It's possible to multithread things, but it would introduce a level of instability that is not good to have in Lithium.
See Jelly's response on #100:
https://github.com/jellysquid3/lithium-fabric/issues/100#issuecomment-670945657
TL;DR: Multithreading isn't a silver bullet, it also has a good chance of breaking vanilla.

**Comment by ghost** [comment]
I put this here because this one is more focused on CPU stuff. Can it be an alpha, like Hydrogen?  (Called Francium because of its instability 🙂 )

**Comment by jellysquid3** [comment]
We're not implementing any kind of world threading in Lithium.
